kaska/smoothn.py

```diff
@@ -2,534 +2,571 @@
 import scipy.optimize.lbfgsb as lbfgsb
```

Comment:
I think it might be best to just revert smoothn. I don't want to have to fix the conflicts here.

Reply:
Done